packages/components/tests/dummy/app/templates/components/pagination-bar.hbs

```diff
@@ -65,33 +135,218 @@
 <section>
   <h3 class="dummy-h3" id="accessibility"><a href="#accessibility" class="dummy-link-section">§</a> Accessibility</h3>
 
-  {{! ADD YOUR CONTENT HERE }}
-
+  <!-- <Doc::WcagList
+    @criteriaList={{array
+      "1.3.1"
+      "1.3.2"
+      "1.4.1"
+      "1.4.3"
+      "1.4.4"
+      "1.4.10"
+      "1.4.11"
+      "1.4.12"
+      "2.1.1"
+      "2.1.2"
+      "2.4.3"
+      "2.4.4"
+      "2.4.6"
+      "2.4.7"
+      "2.5.3"
+      "3.2.1"
+      "4.1.2"
+    }}
+  /> -->
```

Comment:
Commented this out until it's working (was getting an error)

Reply:
Thanks, I just tried merging in the latest main but still saw the error. I will try again later on.